plugins/ern_v0.14.0+/appcenter-analytics_v4.0.2+/AppCenterReactNativeAnalyticsPlugin.java

```diff
@@ -0,0 +1,38 @@
+package com.walmartlabs.ern.container.plugins;
+
+import android.app.Application;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.facebook.react.ReactInstanceManagerBuilder;
+import com.facebook.react.ReactPackage;
+import com.microsoft.appcenter.reactnative.analytics.AppCenterReactNativeAnalyticsPackage;
+
+public class AppCenterReactNativeAnalyticsPlugin implements ReactPlugin<AppCenterReactNativeAnalyticsPlugin.Config> {
+    public ReactPackage hook(@NonNull Application application,
+                             @Nullable Config config) {
+        if (config != null && config.enableInJs != null) {
+            return new AppCenterReactNativeAnalyticsPackage(application, config.enableInJs);
+        }
+        boolean enableDebugMode = false;
+        if (config != null && config.enableDebugMode) {
+            enableDebugMode = true;
+        }
+        return new AppCenterReactNativeAnalyticsPackage(application, enableDebugMode);
```

Comment:
Can you double check the `enableDebugMode` usage here please? It looks like in the constructor this maps to `startEnabled`: https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-analytics/android/src/main/java/com/microsoft/appcenter/reactnative/analytics/AppCenterReactNativeAnalyticsPackage.java#L22

Reply:
It's consistent with the prior configuration. `enableInJs` maps to `startEnabled`

I agree the naming convention requires refactoring to match the open-source terminology in future.

Reply:
> It's consistent with the prior configuration. `enableInJs` maps to `startEnabled`

Hmm with "prior configuration" you mean it was copy+pasted from a different config? In any case, can you please at least open an issue to keep track of it? The naming of the variable is clearly not what's intended, and confusing (specifically talking about "enableDebugMode", not "enableInJs").